src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -169,6 +191,37 @@ object SbtBundle extends AutoPlugin {
     hashArchive
   }
 
+  /**
+   * TODO this is hardcoded to create zip for "frontend" found at src/main/configuration/frontend
+   * TODO maybe this task finds all configurations and creates packages for them?
+   * TODO maybe the user has a way to specify a particular configuration?
+   */
+  private def createConfiguration(): Def.Initialize[Task[File]] = Def.task {
+
+    val name = "frontend"
```

Comment:
We should have a string setting that specifies the configuration to use. Perhaps the default for this setting can simply be "default". How about `configurationName`?